packages/block-editor/src/components/block-lock/toolbar.js

```diff
@@ -60,7 +60,7 @@ export default function BlockLockToolbar( { clientId, wrapperRef } ) {
 			<ToolbarGroup className="block-editor-block-lock-toolbar">
 				<ToolbarButton
 					icon={ lock }
-					label={ __( 'Unlock' ) }
+					label={ __( 'Unlock…' ) }
```

Comment:
Probably not super helpful for block toolbar buttons which are hidden by default.

I also noticed that the "Locking/Unlocking" items are missing the `aria-haspopup/aria-expanded` (#24796). I'll add those tomorrow.

It would be nice to get a11y feedback on this one.

Reply:
@alexstine thoughts on adding an ellipsis for the ToolbarButton text here?